liesel/docs.py

```diff
@@ -0,0 +1,36 @@
+from functools import wraps
+
+
+def usedocs(of: object):
+    """
+    Replaces the docstring of the decorated object with the docstring of the specified
+    object.
+
+    In classes, methods need to be decorated individually. Attribute docstrings are not
+    covered by this decorator.
```

Comment:
Would it make sense to add a decorator that fills all empty method docstrings of a class with the method docstrings from another class? Maybe that would be useful for the kernel protocols and implementations?